OceanComponents/Classes/Components SwiftUI/InlineTextListItem/OceanSwiftUI+InlineTextListItem.swift

```diff
@@ -0,0 +1,229 @@
+            case normal
+            case innactive
+            case positive
+            case warning
+            case highlight
+            case strikethrough
+            case withAction
+        }
+
+        public enum Size {
+            case normal
+            case small
+        }
+    }
+
+    public struct InlineTextListItem: View {
+
+        // MARK: Properties for UIKit
+
+        public lazy var hostingController = UIHostingController(rootView: self)
+        public lazy var uiView = hostingController.getUIView()
+
+        // MARK: Builder
+
+        public typealias Builder = (InlineTextListItem) -> Void
+
+        // MARK: Properties
+
+        @ObservedObject public var parameters: InlineTextListItemParameters
+
+        // MARK: Private properties
+
+        // MARK: Constructors
+
+        public init(parameters: InlineTextListItemParameters = InlineTextListItemParameters()) {
+            self.parameters = parameters
+        }
+
+        public init(builder: Builder) {
+            self.init()
+            builder(self)
+        }
+
+        // MARK: View SwiftUI
+
+        public var body: some View {
+            if parameters.showSkeleton {
+                HStack(alignment: .center, spacing: Ocean.size.spacingStackXs) {
+                    OceanSwiftUI.Skeleton { skeleton in
+                        skeleton.parameters.lines = 1
+                        skeleton.parameters.height = Ocean.size.spacingStackSm
+                    }
+                    OceanSwiftUI.Skeleton { skeleton in
+                        skeleton.parameters.lines = 1
+                        skeleton.parameters.height = Ocean.size.spacingStackSm
+                    }
+                }
+            } else {
+                HStack(alignment: .center, spacing: 0) {
+                    HStack(alignment: .center, spacing: 0) {
+                        if parameters.size == .normal {
+                            OceanSwiftUI.Typography.paragraph { label in
+                                label.parameters.text = parameters.title
+                                label.parameters.multilineTextAlignment = .leading
+                            }
+                        } else {
+                            OceanSwiftUI.Typography.caption { label in
+                                label.parameters.text = parameters.title
+                                label.parameters.multilineTextAlignment = .leading
+                            }
+                        }
+
+                        if let tag = parameters.tag {
+                            OceanSwiftUI.Tag { tagView in
+                                tagView.parameters.label = tag.parameters.label
+                                tagView.parameters.icon = tag.parameters.icon
+                                tagView.parameters.status = tag.parameters.status
+                                tagView.parameters.size = tag.parameters.size
+                            }
+                        }
```

Comment:
Eu acho que isso aqui deve funcionar. Faz um teste

```suggestion
                        if let tag = parameters.tag {
                            tag
                        }
```

ou direto assim. 
```suggestion
                    parameters.tag 
```
Fiz uns testes no badge e ficou bancana... Bem mais limpo
![image](https://github.com/user-attachments/assets/8780056b-bffa-4934-addd-9a0e0f509cde)

![image](https://github.com/user-attachments/assets/e5d6e38e-99c4-4077-bee0-82e33c70c318)

Reply:
acho q nao deveriamos expor o componente por completo em parametros nao, e sim os parameters do componente